app/uk/gov/hmrc/eusubsidycompliancefrontend/services/SubsidyJourney.scala

```diff
@@ -51,10 +51,10 @@ object SubsidyJourney {
   implicit val formPageClaimDateFormat: OFormat[FormPage[DateFormValues]] =
     Json.format[FormPage[DateFormValues]]
 
-
+//added formatting for FormPage[OptionalEORI]
   implicit val formPageOptionalEORIFormat: OFormat[FormPage[OptionalEORI]] =
     Json.format[FormPage[OptionalEORI]]
-
+  //added formatting for FormPage[OptionalTraderRef]
```

Comment:
Won't comment on any more - but there's a bunch of comments that aren't needed.

Reply:
I added those so that you know what was the change i did when I accidently pushed to main 🤦‍♀️  yesterday